### DISCUSSION TITLE
How accurate is the dataset?

### DISCUSSION THREAD

**Discussion by jumpingtom1** [freelawproject/courtlistener#4073] [General]
How accurate is the dataset? Is there any guess as to how many of the opinions are >95% correct?

I encountered this case: https://www.courtlistener.com/opinion/6658574/myers-v-howton/. The text is missing several paragraphs at the beginning and the end. https://scholar.google.com/scholar_case?case=2858089755061678613&q=Myers+v.+Howton,+439+P.3d+472&hl=en&as_sdt=4,38

How common of a problem is this?

**Comment by mlissner** [comment]
This is a very tough question due to the massive scope of the data and the various sources we combine to create our data set. We got this case from the Harvard collection, where you can see it has a similar problem: https://cite.case.law/p3d/439/472/

No collection is perfect, but we fix things as they come up. I've never seen this issue before. 

Can you say more about your use case and maybe we can discuss in more detail?

**Reply by jumpingtom1** [reply]
Interesting - case.law has the full text but coded it improperly. I guess there's nothing to be done.

I'm focusing on state cases for the last 50 years or so - I assume this set of data would be in relatively good shape. I'm using CourtListener API as a source of case text for a gen ai legal analysis project (we emailed earlier this week).
